.github/workflows/automerge-orphans.yml

```diff
@@ -20,14 +20,13 @@ jobs:
           script: |
             const query = `query($owner:String!, $name:String!) {
               repository(owner:$owner, name:$name){
-                pullRequests(first: 100){
+                pullRequests(first: 100, states: OPEN){
```

Comment:
As I see in docs, `states` is array:

![image](https://user-images.githubusercontent.com/20404945/133577590-13dc3549-6701-4f04-90af-2d5c7fe05c91.png)

So probably it should be:

```
pullRequests(first: 100, states: [OPEN]){
```

Reply:
you know it is just an update from master, it is already on master, replicated from .github repo, used for at least 2 weeks, works fine so far https://github.com/asyncapi/generator/runs/3616407238?check_suite_focus=true and anyway I remember I tested in in [GitHub GraphQL Explorer](https://docs.github.com/en/graphql/overview/explorer)

Reply:
If it works, then ok. I didn't do a previous review, so it looks strange for me that field points to array values, and you use single.

Reply:
explorer did not complain 😄

Reply:
`U mnie działa` 😏